_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -452,6 +452,7 @@ mfxStatus VAAPIVideoProcessing::QueryCapabilities(mfxVppCaps& caps)
         if (MFX_FOURCC_NV12    == g_TABLE_SUPPORTED_FOURCC[indx] ||
```

Comment:
At least Query and execute need runtime check if this color format supported by HW.

Reply:
Media driver do not report to application back list of supported to color format.
According to driver's implementation RGBP works from GEN8 (media-driver repo, search for RGBP)